micro-ui/web/packages/modules/workbench/src/components/BoundaryPopup.js

```
@@ -0,0 +1,76 @@
+import { useNavigate } from "react-router-dom";
+
+const BoundaryPopup = ({ showPopUp, setShowPopUp, data, mdmsModuleName }) => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  return (
+    showPopUp && (
+      <PopUp
+        className={"custom-popup-boundary"}
+        type={"default"}
+        heading={t("CHOOSE_MEANS_TO_CREATE_BOUNDARY")}
+        children={[]}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)